fdb-record-layer-jmh/fdb-record-layer-jmh.gradle

```diff
@@ -37,13 +37,28 @@ sourceSets {
 def coreProject = ":${ext.coreProjectName}"
 dependencies {
     api project(coreProject)
+    implementation project(":fdb-record-layer-lucene")
     compileOnly "com.google.code.findbugs:jsr305:${jsr305Version}"
-    testRuntimeOnly "org.apache.logging.log4j:log4j-slf4j-impl:${log4jVersion}"
+
+    testRuntimeOnly("org.apache.logging.log4j:log4j-slf4j-impl:${log4jVersion}") {
+        exclude group: 'org.slf4j', module: 'slf4j-api'
```

Comment:
Out of interest, are you doing print of dependencies? Is that how you found this exclude (or was it just that logging wasn't working?